packages/terra-slide-panel-manager/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "terra-slide-panel-manager",
+  "main": "lib/SlidePanelManager.js",
+  "version": "0.0.0",
+  "description": "A DisclosureManager implementation that presents disclosed content using a SlidePanel.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-slide-panel-manager",
+    "SlidePanelManager",
+    "DisclosureManager",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "devDependencies": {
+    "terra-props-table": "^1.10.1"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^2.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-base": "^2.0.0",
+    "terra-button": "^1.14.0",
+    "terra-content-container": "^1.15.0",
+    "terra-clinical-action-header": "^1.6.0",
+    "terra-app-delegate": "^1.10.0",
+    "terra-disclosure-manager": "^0.x",
```

Comment:
This should also go in peerDependencies too as the same DisclosureManager could be used in conjunction between a ModalManager and SlidePanelManager.